test/invariants/handlers/RewardsManagerHandler.sol

```diff
@@ -69,9 +69,6 @@ contract RewardsManagerHandler is TestBase {
 
   struct GhostRewardPool {
     uint256 totalAssetAmount;
-    uint256 depositSharesAmount;
```

Comment:
I think we should be able to expand the bounds in `boundDepositAmount` now?

Reply:
sorry where / what exactly do you mean here?

Reply:
Sorry -- there is a `boundDepositAmount` function way later in this contract. I think we should be able to expand the minimum and maximum deposit amounts we are using there since we no longer have the receipt token amount calculation/overflow.

Reply:
updated [c573013](https://github.com/Cozy-Finance/cozy-safety-module-rewards-manager/pull/55/commits/c57301328cca804d4ea310d12659ee771b7a5803)